test/client/spec/victory-shared-events/victory-shared-events.spec.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Client tests
+ */
+import { curry, flow, forEach } from "lodash";
+import { mount } from "enzyme";
+import React from "react";
+import { addEvents } from "src/index";
+import VictorySharedEvents from "src/victory-shared-events/victory-shared-events";
+import { MockVictoryComponent, MockDataComponent } from "../mock-components";
+
+describe("components/victory-shared-events", () => {
+  const EventedMockVictoryComponent = addEvents(MockVictoryComponent);
+
+  const findVictoryComponentByName = curry((name, component) => {
```

Comment:
This exists in enzyme as `wrapper.find('[name="one"]')`, and I find it to be much more readable that way. Please use enzyme / sinon / mocha / chai abstraction when possible rather than rolling your own. 

Enzyme selectors: http://airbnb.io/enzyme/docs/api/selector.html